src/components/scenes/GettingStartedScene.tsx

```diff
@@ -101,11 +101,16 @@ export const GettingStartedScene = (props: Props) => {
         swipeOffset.value = 0
       }, 500)
 
-      if (localUsersLength > 0) navigation.navigate('login', { loginUiInitialRoute: 'login-password' })
-      else {
-        logEvent('Signup_Account_Review_Done', {
+      if (localUsersLength > 0) {
+        logEvent('Signup_Welcome', {
           variantId,
-          variantParams: { method: 'swipe' }
+          variantParams: { doneMethod: 'swipe' }
+        })
+        navigation.navigate('login', { loginUiInitialRoute: 'login-password' })
+      } else {
+        logEvent('Signup_Welcome', {
+          variantId,
+          variantParams: { doneMethod: 'swipe' }
         })
```

Comment:
I notice that this exact block of code exists in both branches. You could move it above the `if`, so you only have a single copy. Then the only thing inside the `if` would be the `navigate` call.

Reply:
Good catch, thanks